resources/services/security/security_assessments.go

```diff
@@ -0,0 +1,182 @@
+		Resolver:     fetchSecurityAssessments,
+		Multiplex:    client.SubscriptionMultiplex,
+		DeleteFilter: client.DeleteSubscriptionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"subscription_id", "id"}},
+		Columns: []schema.Column{
+			{
+				Name:        "subscription_id",
+				Description: "Azure subscription id",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAzureSubscription,
+			},
+			{
+				Name:        "display_name",
+				Description: "User friendly display name of the assessment.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.DisplayName"),
+			},
+			{
+				Name:        "code",
+				Description: "Programmatic code for the status of the assessment.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Status.Code"),
+			},
+			{
+				Name:        "cause",
+				Description: "Programmatic code for the cause of the assessment status.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Status.Cause"),
+			},
+			{
+				Name:        "description",
+				Description: "Human readable description of the assessment status.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Status.Description"),
+			},
+			{
+				Name:        "additional_data",
+				Description: "Additional data regarding the assessment.",
+				Type:        schema.TypeJSON,
+				Resolver:    schema.PathResolver("AssessmentProperties.AdditionalData"),
+			},
+			{
+				Name:        "azure_portal_uri",
+				Description: "Link to assessment in Azure Portal.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Links.AzurePortalURI"),
+			},
+			{
+				Name:        "metadata_display_name",
+				Description: "User friendly display name of the assessment.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Metadata.DisplayName"),
+			},
+			{
+				Name:        "metadata_policy_definition_id",
+				Description: "Azure resource ID of the policy definition that turns this assessment calculation on.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Metadata.PolicyDefinitionID"),
+			},
+			{
+				Name:        "metadata_description",
+				Description: "Human readable description of the assessment.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Metadata.Description"),
+			},
+			{
+				Name:        "metadata_remediation_description",
+				Description: "Human readable description of what you should do to mitigate this security issue.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssessmentProperties.Metadata.RemediationDescription"),
+			},
+			{
+				Name:     "metadata_categories",
+				Type:     schema.TypeStringArray,
```

Comment:
Add description

Reply:
Unfortunately there is no description in the SDK. :shrug: